app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= "#{@recipe.name}" %></h1>
+  <%= "#{@recipe.name}" %>
+  <p><%= "Complexity: #{@recipe.complexity} out of 10" %></p>
+  <p><%= "Genre: #{@recipe.genre}" %></p>
+<h3>Ingredients Needed:</h3>
+  <% @ingredients.each do |ingredient| %>
```

Comment:
Here is where instead of creating an `@ingredients` instance variable in your controller, you would just iterate over `@recipe.ingredients.each do`.